R/geom-bar.r

```diff
@@ -40,7 +40,11 @@
 #' automatically determines the orientation from the aesthetic mapping. In the
 #' rare event that this fails it can be given explicitly by setting `orientation`
 #' to either `"x"` or `"y"`. See the *Orientation* section for more detail.
-#' @param width Bar width. By default, set to 90% of the resolution of the data.
+#' @param just Adjustment for column placement. Set to `0.5` by default, meaning
+#'   that columns will be centered about axis breaks. Set to `0` or `1` to place
+#'   columns to the left/right of axis breaks.
+#' @param width Bar width. By default, set to 90% of the `resolution()` of the
```

Comment:
Good idea! How about using ``[`resolution()`]`` to link the document?

Reply:
Sounds good 👍